Library/Homebrew/env_config.rb

```diff
@@ -318,6 +318,11 @@ module EnvConfig
                      "outdated.",
         boolean:     true,
       },
+      HOMEBREW_PIP_INDEX_URL:                    {
+        description:  "A URL string. If set, `brew install <formula>` will use this URL to download PyPI package " \
+                      "resources. If these are multiple URLs separated by spaces, only the first one wil be used.",
```

Comment:
Sorry, again, let's not space separate these and instead only have a single value please.

Reply:
I updated the document. We don't check whether the value is a single URL. But, internally, we ensure that only the first one is used. In the documentation, we ask the user to set a single URL.